JobtechSwe/mydata
# feat(app): add functionality to open consent/login-urls

This makes the app able to handle mydata-urls

**How to test:**

Navigate to https://cv-test.dev.services.jtech.se/ on your iOS-device/simulator, click register, click "Open on this device", you should immediately be asked for consent

-----------

![image](https://user-images.githubusercontent.com/17602389/55793140-7f746300-5ac2-11e9-8402-5b2e584ec869.png)
